website/docs/d/l3_outside.html.markdown

```diff
@@ -10,25 +10,40 @@ description: |-
 # aci_l3_outside #
 Data source for ACI L3 Outside
 
+## API Information ##
+
+* `Class` - l3extOut
+* `Distinguished Name` - uni/tn-{tenant_name}/out-{l3_outside_name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Networking -> L3Outs
+
 ## Example Usage ##
 
 ```hcl
-data "aci_l3_outside" "dev_l3_out" {
-  tenant_dn  = aci_tenant.dev_tenant.id 
-  name       = "foo_l3_out"
+data "aci_l3_outside" "foo_l3_outside" {
+  tenant_dn = aci_tenant.terraform_tenant.id
+  name      = "foo_l3_outside"
 }
 ```
-## Argument Reference ##
-* `tenant_dn` - (Required) Distinguished name of parent Tenant object.
-* `name` - (Required) name of Object l3 outside.
-
 
+## Argument Reference ##
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the L3 Outside object.
 
 ## Attribute Reference
 
 * `id` - Attribute id set to the Dn of the L3 Outside.
-* `description`- (Optional) Description for object l3 outside.
-* `annotation` - (Optional) annotation for object l3 outside.
-* `enforce_rtctrl` - (Optional) enforce route control type
-* `name_alias` - (Optional) name_alias for object l3 outside.
-* `target_dscp` - (Optional) The target differentiated services code point (DSCP) of the path attached to the layer 3 outside profile.
+* `description`- (Optional) Description of the L3 Outside object.
+* `annotation` - (Optional) Annotation of the L3 Outside object.
+* `enforce_rtctrl` - (Optional) Enforce route control type of the L3 Outside object. 
+* `name_alias` - (Optional) Name alias of the L3 Outside object.
+* `target_dscp` - (Optional) The target differentiated services code point (DSCP) of the path attached to the L3 Outside object.
+* `relation_l3ext_rs_dampening_pol` - (Optional) Relation to Route Control Profile for Dampening Policies (class rtctrlProfile). Cardinality - N_TO_M. Type - Block.
+  * tn_rtctrl_profile_name - (Deprecated) Name of the Route Control Profile for Dampening Policies.
+  * tn_rtctrl_profile_dn - (Optional) Distinguished name of the Route Control Profile for Dampening Policies.
+  * af - (Optional) Address Family of the Dampening Policies.
+* `relation_l3ext_rs_ectx` - (Optional) Relation to VRF (class fvCtx).
```

Comment:
should we include cardinality and extra description for these as is done in the resource?